draft-ietf-secevent-subject-identifiers.md

```diff
@@ -265,6 +271,19 @@ Below is a non-normative example Subject Identifier in the Opaque Identifier For
 ~~~
 {: #figexamplesubidopaque title="Example: Subject Identifier in the Opaque Identifier Format"}
 
+### Decentralized Identifier (DID) Format {#sub-id-did}
+The Decentralized Identifier Format identifies a subject using a Decentralized Identifier (DID) URL, identified with a DID URL as defined in {{DID}}.  Subject Identifiers in this format MUST contain a `uri` member whose value is the DID URL for the subject. The value of the `uri` member MAY be any valid DID URL including a bare DID. The `uri` member is REQUIRED and MUST NOT be null or empty. The Decentralized Identifier Format is identified by the name `did`.
```

Comment:
1. Replaced `uri` with `url`, since these are DID UR *L* s. Could be convinced to use `uri` though, if there's a good reason. (e.g., existing convention within the DID community)
2. The "MAY" was confusing to me; tweaked the language to be a bit clearer (IMHO; happy to iterate on it if needed)

```suggestion
### Decentralized Identifier (DID) Format {#sub-id-did}
The Decentralized Identifier Format identifies a subject using a Decentralized Identifier (DID) URL as defined in {{DID}}.  Subject Identifiers in this format MUST contain a `url` member whose value is the DID URL for the subject. The value of the `url` member MUST be a valid DID URL as defined in {{DID}}, and MAY be a bare DID. The `url` member is REQUIRED and MUST NOT be null or empty. The Decentralized Identifier Format is identified by the name `did`.
```

Reply:
Should this be restricted to *absolute* DID URLs? We should specify, either way. Maybe "absolute only" is implied, but it seems worth clarifying since did-core talks about relative DID URLs.

Reply:
> the DID URL for the subject

Should this be "*a* DID URL for the subject"? Can a subject have more than one DID URL, or do different DID URLs necessarily refer to different subjects?